packages/addons/src/plugins/localStoragePlugin.ts

```diff
@@ -41,6 +41,9 @@ export function createLocalStoragePlugin(
     // to 'form' and 'group' inputs — as well as any add-on inputs
     // registered of FormKit type 'group' (eg. 'multi-step').
     if (node.type !== 'group') return
+    
+    // enable SSR support
+    if (typeof window === "undefined" || !window.localStorage) return
```

Comment:
```suggestion
    if (typeof window === "undefined") return
```
Happy to merge this into the next release! We can probably just bail if the window is undefined as `.localStorage` is going to exist on any browser FormKit supports.

Reply:
Sounds good to me, thanks! Force pushed an update.